rust/Cargo.toml

```diff
@@ -60,6 +60,7 @@ lossy_float_literal = "warn"
 result_large_err = "allow"
 
 # Example: how to point modeling-app at a different repo (e.g. a branch or a local clone)
-#[patch.crates-io]
-#kittycad-modeling-cmds = { path = "../../../modeling-api/modeling-cmds" }
+[patch.crates-io]
+kittycad-modeling-cmds = { git = "https://github.com/KittyCAD/modeling-api.git", branch = "ben/conics" }
+# kittycad-modeling-cmds = { path = "../../modeling-api/modeling-cmds" }
```

Comment:
I tried to merge `main`, but I ended up with build errors. I'm guessing your modeling-api branch needs updating since I just made a release of that and updated it here.